resultbrowser/src/css/default.css

```diff
@@ -50,11 +51,11 @@ body {
  */
 #progressmeter {
     display: none;
-    background-image: url(data-url:../images/progress.gif);
-    width: 24px;
-    height: 24px;
-    float: right;
-    margin: 2px;
+    margin: 10px;
+    position: fixed;
+    top: 0;
+    right: 0;
+    z-index: 2147483647;
```

Comment:
As we have full control over our CSS, there is no need to use such a high value for z-index. IMO, a value of 10 should be sufficient.

Reply:
Done